### PR TITLE
Rookout add resources requests

### PR DESCRIPTION
**Pull request by MickaelAlliel**
#### What this PR does / why we need it:
This PR adds resource requests and limits and fixes typos/naming and links

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by Zohar2506** [comment]
I approve @MickaelAlliel is maintainer and owner of this helm chart from our organization.

File: stable/rookout/README.md [hunk 1]
```diff
@@ -42,25 +42,31 @@ The command removes all the Kubernetes components associated with the chart and
...
+
+
+The above parameters map to the env variables defined in [rookout/agent](https://docs.rookout.com/docs/agent-setup.html). For more information please refer to the [rookout/agent](https://hub.docker.com/r/rookout/agent/) image documentation.
 
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
 
 ```bash
 $ helm install --name my-release \
-  --set token=YOUR_TOKEN_HERE,listenAll=False,tags=tag1;tag2;tag3 \
+  --set rookout.token=YOUR_ORGANIZATIONAL_TOKEN,listenAll=False,tags=tag1;tag2;tag3 \
```
Comment (Zohar2506) [inline stable/rookout/README.md]: Remove tags from agent

File: stable/rookout/values.yaml [hunk 2]
```diff
@@ -1,17 +1,36 @@
...
   pullPolicy: IfNotPresent
 
 service:
   type: ClusterIP
 
-listenAll: True
-tags: ''
-
 rookout:
+  listenAll: True
```
Comment (Zohar2506) [inline stable/rookout/values.yaml]: Set false as default

**Comment by MickaelAlliel** [comment]
Still needs approval...

File: stable/rookout/OWNERS [hunk 3]
```diff
@@ -0,0 +1,10 @@
+approvers:
+- MickaelAlliel
+- itielshwartz
+- LiranLast
+- Zohar2506
+reviewers:
+- MickaelAlliel
+- itielshwartz
+- LiranLast
+- Zohar2506
```
Comment (mattfarina) [inline stable/rookout/OWNERS:10]: Each of these people will need to follow the process at https://github.com/helm/charts#owning-and-maintaining-a-chart. Of the 4 people listed there are a total of 2 commits to charts.

Also, each person should be listed as a maintainer in the `Chart.yaml`.

**Review by mattfarina — COMMENTED** [review]
If you would like to have more control over the commit process you can host the chart yourself and have it listed on https://hub.helm.sh. For example, you can [host a chart repo on GitHub pages](https://github.com/helm/hub/blob/master/docs/github.md). Then you can control the permissions and rate of publication.

**Pull request closed without merging**